pnpm-workspace.yaml

```diff
@@ -4,11 +4,12 @@ packages:
   - tooling/*
 
 catalog:
-  "@tanstack/react-query": ^5.59.15
-  "@trpc/client": ^11.0.0-rc.477
-  "@trpc/react-query": ^11.0.0-rc.477
-  "@trpc/server": ^11.0.0-rc.477
-  eslint: ^9.12.0
+  "@tanstack/react-query": ^5.59.16
+  "@trpc/client": &trpcVersion ^11.0.0-rc.601
+  "@trpc/react-query": *trpcVersion
+  "@trpc/server": *trpcVersion
```

Comment:
ehh idk... i get the idea but it just looks weird... i prefer just separating groups with a newline or something to make it clear they're related?

Reply:
> ehh idk... i get the idea but it just looks weird... i prefer just separating groups with a newline or something to make it clear they're related?

Something like this?

```yaml
```